lib/cylc/job_pool.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python2
```

Comment:
Shebangs may be gone later (see #3041), but for now I think this one would be `python3`?